ssl/ssl_cert_comp.c

```diff
@@ -101,24 +101,39 @@ __owur static OSSL_COMP_CERT *OSSL_COMP_CERT_from_uncompressed_data(unsigned cha
         method = COMP_brotli_oneshot();
         break;
     case TLSEXT_comp_cert_zlib:
-        method = COMP_zlib();
+        method = COMP_zlib_oneshot();
         break;
     case TLSEXT_comp_cert_zstd:
         method = COMP_zstd_oneshot();
         break;
     default:
+        fprintf(stderr, "COMP ERROR LINE=%d\n", __LINE__);
```

Comment:
__LINE__ is not used in many places in OpenSSL. I can see there is a OPENSSL_LINE in macros.h.

Reply:
@tmshort fprintf to stderr is not a good idea for error reporting in libssl. Either use trace api if this is rather tracing thing or ERR_raise_data.

Reply:
This was my debugging that I missed! It should not be there.